codegens/python-requests/test/unit/converter.test.js

```diff
@@ -153,4 +153,31 @@ describe('Python- Requests converter', function () {
       .to.throw('Python-Requests~convert: Callback is not a function');
   });
 
+  it('should not have allow_redirects=False twice in generated snippet when' +
+  ' followRedirect option is set as false', function () {
+    var request = new sdk.Request(mainCollection.item[0].request),
+      options = { followRedirect: false };
+    convert(request, options, function (err, snippet) {
+      if (err) {
+        expect.fail(null, null, err);
+      }
+      expect(snippet).to.be.a('string');
+      expect(snippet).to.not.include('allow_redirects=False, allow_redirects=false');
```

Comment:
If `options.requestTimeout !== 0` is true than this will be not valid as generated snippet will have `allow_redirects=False, timeout=${options.requestTimeout}, allow_redirects=false`.